packages/graphql/src/schema/types/objects/CartesianPoint.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GraphQLFloat, GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from "graphql";
+
+export const CartesianPoint = new GraphQLObjectType({
+    name: "CartesianPoint",
+    fields: {
+        x: {
+            type: new GraphQLNonNull(GraphQLFloat),
+            resolve: (source) => {
+                return source.point.x;
+            },
```

Comment:
**question (non-blocking)**: I'm honestly not sure what the reasoning/advantage of using arrow functions here is :confused:. It feels like the function syntax is equivalent, slightly shorter and easier to read?

Happy to keep it this way if you disagree
```suggestion
            resolve(source) {
                return source.point.x;
            },
```